modules/caddyhttp/templates/tplcontext.go

```diff
@@ -398,6 +401,41 @@ func (c TemplateContext) funcHTTPError(statusCode int) (bool, error) {
 	return false, caddyhttp.Error(statusCode, nil)
 }
 
+// formatHumane returns a human readable format of some data
+// For time formating is the default format RFC1123Z.
+// on this page are the definitions of that format https://pkg.go.dev/time#pkg-constants
+// Example usage:
+// ```
+// {{formatHuman "size" "2048000"}}
+// {{placeholder "http.response.header.content-length" | formatHuman "size"}}
+// {{formatHuman "time" "Fri, 05 May 2022 15:04:05 +0200"}}
+// ```
```

Comment:
You can add this example to the `templates.go` file, there's a big godoc block at the top; that godoc is what gets turned into the docs on the website: https://caddyserver.com/docs/modules/http.handlers.templates#docs